actors/builtin/power/power_test.go

```diff
@@ -808,6 +976,39 @@ func (h *spActorHarness) constructAndVerify(rt *mock.Runtime) {
 	verifyEmptyMap(h.t, rt, st.CronEventQueue)
 }
 
+type confirmedSectorSend struct {
+	miner      addr.Address
+	sectorNums []int
```

Comment:
Just use `abi.SectorNumber`; you shouldn't need to cast when initialising as `[]abi.SectorNumber{1, 2, 3}`.